tests/unit/amo/pages/TestUsersUnsubscribe.js

```diff
@@ -61,43 +51,36 @@ describe(__filename, () => {
     );
   };
 
-  it('renders loading indicators when the user is not unsubscribed yet', () => {
-    const root = render();
-
-    expect(root.find(Card)).toHaveLength(1);
-    expect(root.find(Card)).toHaveProp('header', <LoadingText />);
-
-    expect(
-      root.find('.UsersUnsubscribe-content-explanation').find(LoadingText),
-    ).toHaveLength(1);
+  const createErrorHandlerId = ({ params }) => {
+    return `src/amo/pages/UsersUnsubscribe/index.js-${extractId({
+      match: { params },
+    })}`;
+  };
 
-    expect(
-      root.find('.UsersUnsubscribe-content-notification').find(LoadingText),
-    ).toHaveLength(1);
+  it('renders loading indicators when the user is not unsubscribed yet', () => {
+    render();
 
-    expect(
-      root.find('.UsersUnsubscribe-content-edit-profile').find(LoadingText),
-    ).toHaveLength(1);
+    expect(screen.getAllByRole('alert')).toHaveLength(4);
   });
 
-  it('renders an HTML title', () => {
-    const root = render();
+  it('renders an HTML title', async () => {
+    render();
+
+    await waitFor(() => expect(getElement('title')).toBeInTheDocument());
 
-    expect(root.find('title')).toHaveLength(1);
-    expect(root.find('title')).toHaveText('Unsubscribe');
+    expect(getElement('title')).toHaveTextContent('Unsubscribe');
   });
 
   it('dispatches unsubscribeNotification on mount', () => {
     const params = getParams();
     const { store } = dispatchClientMetadata();
```

Comment:
I have adopted a pattern for most of the test suites where we `let store` at the top, and then create the `store` in `beforeEach`, so that there is always a `store` available to the tests, and it is the same `store` that is used in `render`, so we don't need to pass it around. I find this cleaner, and it might be best, for consistency sake, to do this in this test suite as well.